weave

```diff
@@ -801,6 +857,7 @@ case "$COMMAND" in
     expose)
         collect_cidr_args "$@"
         shift $CIDR_COUNT
+        [ $CIDR_COUNT -gt 0 ] || { get_ipam_cidr weave && expose $CIDR_ARGS ; }
```

Comment:
Surely we need a corresponding change to 'hide' in order to release the IP address.